CHANGELOG.md

```diff
@@ -86,6 +86,17 @@
  - #421 Add support for authLoginDomain in the Redfish.conf
  - #370 Use a label to save template_UUID instead of Description attribute in the server profile
  - #422 Save OneView Python SDK client for multiple requests
```

Comment:
It is missing the issues of [sprint 25](https://github.com/HewlettPackard/oneview-redfish-toolkit/milestone/26?closed=1), [sprint 24](https://github.com/HewlettPackard/oneview-redfish-toolkit/milestone/25?closed=1) ou [sprint 23](https://github.com/HewlettPackard/oneview-redfish-toolkit/milestone/24?closed=1).
I think we had forgot put before. :sweat: